static/js/components/widgets/ResourcePickerDialog.tsx

```diff
@@ -113,6 +113,13 @@ export default function ResourcePickerDialog(props: Props): JSX.Element {
       .filter(entry => entry.category === "Content")
       .flatMap(elt => [elt.name, elt.label])
```

Comment:
Not really related to the bug, but IMO  we should just use the name, not name or label.